config/routes.rb

```diff
@@ -2,6 +2,7 @@
   get 'welcome/index'
 
 root 'welcome#index'
+get 'welcome/abc'
```

Comment:
get 'welcome/abc' will create a route '/welcome/abc', lekin ye kon se controller ke kon se action mein jayega ye specified nahi h.
ye google kr
list all routes in rails